rules/Php80/Rector/NotIdentical/StrContainsRector.php

```diff
@@ -82,6 +85,14 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
+        if (isset($funcCall->args[2])) {
+            if ($this->isName($funcCall->name, 'strpos') && $this->isPositiveInteger($funcCall->args[2]->value)) {
+                $subStrFuncCall = new FuncCall(new Name('substr'), [$funcCall->args[0], $funcCall->args[2]]);
+                $funcCall->args[0] = new Arg($subStrFuncCall);
```

Comment:
You can use nodeFactory service for it:

```suggestion
                $funcCall->args[0] = new Arg($this->nodeFactory->createFuncCall('substr', [$funcCall->args[0], $funcCall->args[2]]));
```